docs/style-guide.rst

```diff
@@ -1058,14 +1058,40 @@ Inside each contract, library or interface, use the following order:
 1. Type declarations
 2. State variables
 3. Events
-4. Modifiers
-5. Functions
+4. Errors
+5. Modifiers
+6. Functions
 
 .. note::
 
     It might be clearer to declare types close to their use in events or state
     variables.
 
+Yes:
+
+.. code-block:: solidity
+
+    // SPDX-License-Identifier: GPL-3.0
+    pragma solidity >=0.8.0 <0.9.0;
+
+    abstract contract Math {
+        error DivideByZero();
+        function divide(int256 numerator, int256 denominator) public virtual returns (uint256);
+    }
+
+No:
+
+.. code-block:: solidity
+
+    // SPDX-License-Identifier: GPL-3.0
+    pragma solidity >=0.8.0 <0.9.0;
```

Comment:
By the way, custom errors were only introduced in `0.8.4`. I'll push the required changes.